app/models/submission.rb

```diff
@@ -14,5 +14,9 @@ class Submission < ApplicationRecord
     state :in_review
     state :completed
     state :rejected
+
+    event :ready_for_review do
+      transitions from: %i[pending processing], to: :in_review
```

Comment:
ah, yeah I did this as a temporary measure as at the moment I don't believe there is nothing that transitions submissions from their initial "pending" state to a "processing" state. The plan is to update the ingest script to do that, but until we do it would be good to see the transitions firing.

Reply:
Fair enough :+1:

Reply:
I've rebased and also reworded the commit message to explain this!